backend/apps/core/management/commands/seed_companies.py

```
@@ -0,0 +1,467 @@
+        companies = []
+        csv_file = Path(csv_path)
+        
+        if not csv_file.exists():
+            raise FileNotFoundError(f"CSV file not found: {csv_path}")
+        
+        # Try UTF-8 with BOM first, then fallback to UTF-8
+        encodings = ['utf-8-sig', 'utf-8']
+        
+        for encoding in encodings:
+            try:
+                with open(csv_file, 'r', encoding=encoding) as f:
```

Ruff (0.14.3)

Unnecessary mode argument

Remove mode argument

(UP015)